docs/rules.md

```diff
@@ -58,6 +58,7 @@
 * [no-whitespace-for-layout](rule/no-whitespace-for-layout.md)
 * [no-whitespace-within-word](rule/no-whitespace-within-word.md)
 * [no-yield-only](rule/no-yield-only.md)
+* [no-yield-to-default](rule/no-yield-to-default.md)
```

Comment:
This file has been deleted, please rebase on the latest master and run `yarn update` to update the README.